bokehjs/src/coffee/common/embed.coffee

```diff
@@ -117,7 +117,19 @@ fill_render_item_from_script_tag = (script, item) ->
 
   logger.info("Will inject Bokeh script tag with params #{JSON.stringify(item)}")
 
-embed_items = (docs_json, render_items, websocket_url) ->
+embed_items = (docs_json_or_id, render_items, websocket_url) ->
```

Comment:
if we end up with both externalized docs_json and in-script docs_json, let's make two separate functions here in `Bokeh.embed` rather than type-introspecting the parameter...